### ISSUE TITLE
Add a report filter by parent class or interface

### ISSUE THREAD

**Issue by shanshin** [Kotlin/kotlinx-kover#454]
In some cases, it is convenient to exclude from the report all classes inherited from one parent, or implementing a specific interface.
This is more convenient than annotating several classes located in different packages, but inherited from the same parent.

A new filter type should be added to the DSL:
```
koverReport {
    filters {
        excludes {
            inheritedFrom("foo.ParentClass", "foo.Interface")
        }        
    }
}
```

The possibility of this filtering should be added to `intellij-reporter`.

**Comment by zuevmaxim** [comment]
https://youtrack.jetbrains.com/issue/IDEA-338719

**Comment by shanshin** [comment]
Implemented in `0.8.0`
